daemon/docker.go

```diff
@@ -17,9 +17,15 @@ import (
 	git "gopkg.in/src-d/go-git.v4"
 )
 
+var projectName = ""
```

Comment:
I feel that this could use a default of some sort (perhaps `"project"`), for no real reason other than this looks a bit strange 😅

Reply:
done